docs/releasenotes/unreleased/template.jinja

```diff
@@ -36,7 +36,8 @@ Rule changes
 {% if fixes|length > 0 %}
 Fixes
 =====
```

Comment:
Would be good to add an extra line after each header. Look at the generated file that it's missing there.